.dependabot/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
+
+update_configs:
+   - package_manager: "javascript"
```

Comment:
I guess we want npm here?

Reply:
I beleive javascript means updating javascript dependencies. In this way dependabot will take a look at package.json and package-lock.json. This is the only way to select it and one cannot specify npm.